packages/stream_chat_flutter/lib/src/message_input/attachment_picker/stream_attachment_picker_bottom_sheet.dart

```diff
@@ -115,7 +116,7 @@ Future<T?> showStreamAttachmentPickerModalBottomSheet<T>({
               currentPlatform == TargetPlatform.linux ||
               currentPlatform == TargetPlatform.windows;
 
-          if (isWebOrDesktop) {
+          if (isWebOrDesktop || useNativeAttachmentPickerOnMobile) {
             return webOrDesktopAttachmentPickerBuilder.call(
```

Comment:
Maybe we should rename this now `webOrDesktopAttachmentPickerBuilder`? It's pretty confusing looking at the code why it works for native mobile scenarios also, as the whole flow from this point only refers to `desktop` and `web`. 
Maybe `nativeAttachmentPickerBuilder` would be better?

Reply:
I agree to the `nativeAttachmentPickerBuilder` name but there's several other associated config objects that I would not be sure how to rename. We can discuss this.

Reply:
Maybe a comment linking to the docs for this with more info would be helpful here. Easy to get an impression that everything native is handled automatically by the SDK.